mdpow/templates/system.top

```diff
@@ -10,7 +10,6 @@
 #include "oplsaa.ff/forcefield.itp"
 
 ; Include compound topology
-#include "compound.prm"
```

Comment:
@VOD555 are you sure that this belongs in this PR?

Reply:
It's the change in a previous pr about adding .prm files. This line would have some problem for current tests, but I didn't mention that due to the environment problem. So for this PR I think removing this line is ok. In the #103 , I'll add this line back.